api/bin/spark.js

```diff
@@ -27,7 +27,7 @@ assert(DEAL_INGESTER_TOKEN, 'DEAL_INGESTER_TOKEN is required')
 const client = new pg.Pool({
   connectionString: DATABASE_URL,
   // allow the pool to close all connections and become empty
-  min: 0,
+  // min: 0,
```

Comment:
I am curious - why do we need this change? Can you please add a code comment to capture the reasoning? 

Should we update all other Spark services communicating with PG to use the same settings, too?

Reply:
Great catch, I must've committed this accidentally.